web/third/bootstrap/css/bootstrap-btn.css

```diff
@@ -131,47 +131,12 @@ fieldset[disabled] .btn-default.active {
   background-color: #3276b1;
   border-color: #285e8e;
 }
-.btn-success {
-  color: #fff;
```

Comment:
I am not sure whether we should remove this particular CSS for `.btn-success` or not. I think this will lead to some unexpected changes. But again I cannot test it as it is used on billing page.

Reply:
I've posted a couple gif comparsions of before and after in the PR description. Doesn't seem like there is any unexpected changes since the color was overwritten by another css rule as well.

Reply:
Ok, can you tell which CSS overrides the color? I cannot find any.

Reply:
The class is only used in templates/corporate/billing.html where the style is overwritten by https://github.com/zulip/zulip/blob/fe54df0b01569f2442bfe09800d8398d25ae36af/web/styles/portico/landing_page.css#L144

Reply:
Ok, cool. I am not sure what styling do we want. Seems fine for now as we do not change the original styling in this PR.

Reply:
I posted in a new thread in #frontend about removing these buttons styles, but that CSS rule does seem to have an (ugly) effect on that button.